pandas/tests/frame/test_missing.py

```diff
@@ -257,6 +257,20 @@ def test_fillna(self):
         result = df.fillna(value={'Date': df['Date2']})
         assert_frame_equal(result, expected)
 
+        # with timezone
+        # GH 15855
+        df = pd.DataFrame({'A': [pd.Timestamp('2012-11-11 00:00:00+01:00'),
+                                 pd.NaT]})
+        exp = pd.DataFrame({'A': [pd.Timestamp('2012-11-11 00:00:00+01:00'),
+                                  pd.Timestamp('2012-11-11 00:00:00+01:00')]})
+        self.assert_frame_equal(df.fillna(method='pad'), exp)
```

Comment:
try to not use ``self.`` here, just ``assert_frame_equal`` (or ``tm.assert_frame_equal``).